README.md

```diff
@@ -65,15 +65,31 @@ This will create a libvirt based virtual machine running the Katello server on C
 
 ### Development Deployment
 
-A Katello development environment can be deployed on CentOS 6 or 7. Ensure that you have followed the steps to setup Vagrant and the libvirt plugin.
+A Katello development environment can be deployed on CentOS 6 or 7. Ensure that you have followed the steps to setup Vagrant and the libvirt plugin. There are a variety of useful development environment options that should or can be set when creating a development box. These options are designed to configure your environment ready to use your own fork, and create pull requests. To create a development box:
 
-To deploy to CentOS 6:
+  1. Copy boxes.yaml.example to boxes.yaml. If you already have a boxes.yaml, you can copy the entries in boxes.yaml.example to your boxes.yaml.
+  2. Now, replace <my_github_username> with your github username
+  3. Fill in any other options, examples:
```

Comment:
Maybe say something about the other options available or where to find them, such as [puppet-katello_devel](https://github.com/Katello/puppet-katello_devel/blob/master/manifests/init.pp)?